### DISCUSSION TITLE
Any plans to update this?

### DISCUSSION THREAD

**Discussion by dafunks** [5x/easy-steam-free-packages#11] [General]
I always find myself checking back here to see if any updates have happened with new games or software. No updates for a few years now. Any plans to add anything? 

Thanks for the work so far as its given a lot to my account.

**Comment by louisa-uno** [comment]
Hi @dafunks,
I'm currently developing a script which fetches all free steam apps.
I will add an updated script to this repository as soon as I'm finished with it.

**Reply by louisa-uno** [reply]
@dafunks I've created a [fork](https://github.com/Luois45/easy-steam-free-packages/blob/master/dist/easysfp.js) with an updated script and made a pull request.
It does contain roughly 24k packages.

**Reply by dafunks** [reply]
Thank you so much. 

I wonder if these scripts could be turned into a small app or program that could just be ran in the background on my pc or phone. Just so save having a window open and one that could just update easily with new games or software. 

just a thought I had. It may not be possible and even if so I do not have the knowhow.

**Reply by louisa-uno** [reply]
@dafunks 
I took a quick look into it and do think that the best solution for it would be using the the command and program discussed [here](https://steamcommunity.com/groups/archiasf/discussions/1/2265817650395227391/#c2265817650395938645).

If I do have enough time this week I'll definitely try to build a little application which is capable of running in the background and does maybe fetch the games from a GitHub repository which gets updated automatically with GitHub Actions or so.

One problem I do encounter with owning a ton of steam games in general: The scroll bar in your steam library isn't usable at all anymore because it lags severely.

**Reply by dafunks** [reply]
Yeah this is something I have too. I did read a steam forum about something similar to this that put games and software into categories automatically based on some tags. I actually looked for it around the time I found this and I couldn't find it. The post was very old and I am guessing Steam works differently now.

**Reply by louisa-uno** [reply]
@dafunks The [script](https://github.com/Luois45/claim-free-steam-packages) I made is now finished.
